package-lock.json

```diff
@@ -17098,7 +17098,7 @@
     "filter-obj": {
       "version": "1.1.0",
       "resolved": "https://registry.npmjs.org/filter-obj/-/filter-obj-1.1.0.tgz",
-      "integrity": "sha512-8rXg1ZnX7xzy2NGDVkBVaAy+lSlPNwad13BtgSlLuxfIslyt5Vg64U7tFcCt4WS1R0hvtnQybT/IyCkGZ3DpXQ=="
+      "integrity": "sha1-mzERErxsYSehbgFsbF1/GeCAXFs="
```

Comment:
Is this intentional? Thought It was strange to only change this one line.

Reply:
oh no, it's not

Reply:
Can we revert this?

Reply:
For sure, I'll do it when I receive all review comments